pkgs/applications/audio/midi-trigger/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, gnumake, pkg-config, gcc, lv2 }:
+stdenv.mkDerivation rec {
+  pname = "midi-trigger";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "unclechu";
+    repo = "MIDI-Trigger";
+    rev = "v${version}";
+    sha256 = "sha256-tMnN8mTd6Bm46ZIDy0JPSVe77xCZws2XwQLQexDWPgU=";
+  };
+
+  nativeBuildInputs = [ gnumake pkg-config gcc ];
```

Comment:
gnumake and gcc should be part of stdenv and don't need to be explicitly included.

Reply:
Removed those.

Reply:
@K900 Darwin builder failed with:
```
       > patching sources
       > configuring
       > no configure script, doing nothing
       > building
       > build flags: SHELL=/nix/store/wns6kkf93d55703pssdwz2dj0ws5pfvf-bash-5.1-p16/bin/bash BUILD_DIR=.
       > mkdir --parent -- "." "."/interface
       > gcc -std=c99 -fPIC -DPIC src/rms.c -c -o "."/rms.o
       > /nix/store/wns6kkf93d55703pssdwz2dj0ws5pfvf-bash-5.1-p16/bin/bash: line 1: gcc: command not found
       > make: *** [Makefile:43: rms.o] Error 127
       For full logs, run 'nix log /nix/store/bbwv7cxjm395z033g8kak5hbgzyrwh3a-midi-trigger-0.0.4.drv'.
```

See https://github.com/NixOS/nixpkgs/pull/182015/checks?check_run_id=7437688907

I think it’s because `clang` is the default for macs? So it seems it doesn’t hold true that `stdenv` always includes GCC?

Reply:
Does it have to use GCC specifically? Try using `$CC` instead.

Reply:
@K900 This would require to make a new release for the app. I didn’t plan to do this yet. I guess I can make an override of the compiler depending on the platform in `makeFlags`?

Reply:
@K900 is there a guarantee that for Darwin there always be Clang provided by `stdenv`?

Reply:
Depending on just gcc is generally not a good idea for nixpkgs. Many stdenvs use other compilers intentionally.

Reply:
Also, if you're going to cut a new release, maybe add an `install` target to the makefile? Or just use an existing build system like Meson or CMake.

Reply:
@K900 I’ll use your notes when I get back to making a new release. Will add `install` command as well.

Reply:
I pushed an override to Clang in `makeFlags` for Darwin platform.